src/mlpack/methods/range_search/rs_model_impl.hpp

```diff
@@ -18,6 +18,216 @@
 namespace mlpack {
 namespace range {
 
+//! Monochromatic range search on the given RSType instance.
+template<typename RSType>
+void MonoSearchVisitor::operator()(RSType *rs) const
+{
+  if (rs)
+    return rs->Search(range, neighbors, distances);
+  throw std::runtime_error("no range search model initialized");
+}
+
+//! Save parameters for bichromatic range search.
+BiSearchVisitor::BiSearchVisitor(const arma::mat& querySet,
+                                                        const math::Range& range,
+                                                        std::vector<std::vector<size_t>>& neighbors,
+                                                        std::vector<std::vector<double>>& distances,
+                                                        const size_t leafSize
+                                                        ) :
```

Comment:
Looks like something strange happened with the spacing here; be sure your editor is using spaces and not tab characters.  (I think we really need to get an automatic style checker running for PRs, then there will be fewer random comments from me! :))